app/javascript/components/dashboard/routes/AnalyticsRoute.tsx

```
@@ -0,0 +1,80 @@
+
+  if (error) {
+    return <ErrorFallback error={new Error(error)} />;
+  }
+
+  if (isLoadingAnalytics && !analyticsData) {
+    return <PageLoading message="Loading analytics..." />;
+  }
+
+  // For now, redirect to the existing analytics page
+  // TODO: Create a React-based analytics component
+  useEffect(() => {
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)